OrbitClientServices/TracepointServiceClient.cpp

```
@@ -14,30 +14,29 @@ using orbit_grpc_protos::TracepointService;
 TracepointServiceClient::TracepointServiceClient(const std::shared_ptr<grpc::Channel>& channel)
     : tracepoint_service_(TracepointService::NewStub(channel)) {}
 
-void TracepointServiceClient::PopulateWithServerData() {
+std::unique_ptr<grpc::ClientContext> TracepointServiceClient::CreateContext() const {
+  auto context = std::make_unique<grpc::ClientContext>();
+
+  return context;
+}
+
+std::vector<TracepointInfo> TracepointServiceClient::GetTracepointList() const {
   GetTracepointListRequest request;
   GetTracepointListResponse response;
   std::unique_ptr<grpc::ClientContext> context = CreateContext();
 
+  std::vector<TracepointInfo> tracepoint_list;
   grpc::Status status = tracepoint_service_->GetTracepointList(context.get(), request, &response);
   if (!status.ok()) {
     ERROR("gRPC call to GetTracepointList failed: %s (error_code=%d)", status.error_message(),
           status.error_code());
```

You probably still want to return a error here if the call has failed.

---

you can also do ERROR(error_message);